config/jobs/kubernetes/sig-testing/verify.yaml

```diff
@@ -26,21 +28,20 @@ presubmits:
         resources:
           requests:
             cpu: 4
-
   # TODO(krzyzacy): add branches once this works
   # TODO(krzyzacy): Consider combine kubekins-e2e and kubekins-test
-  - name: pull-kubernetes-verify-podutil
+  - name: pull-kubernetes-verify
     always_run: false
     decorate: true
-    branch:
```

Comment:
I love YAML sooo much

Reply:
we could switch to strict parsing, and we should